pystachio/__init__.py

```diff
@@ -15,6 +15,9 @@
     Float,
     Integer,
     String)
+from .choice import (
+   Choice
```

Comment:
Please stick to local style: 4 space hanging indent, closing paren on same line as last import list element.
That said - this could probably just be 1 line with no parens like `from .base import Environment` above.